.travis_dev.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js: "8"
+script:
+  - yarn lint
+  - yarn test
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "staging" ]); then yarn run build; fi
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+cache:
+  yarn: true
+  directories:
+  - node_modules
+deploy:
+  - provider: s3
+    access_key_id: $AWS_ACCESS_KEY_ID
+    secret_access_key: $AWS_SECRET_ACCESS_KEY
+    bucket: cru-give-web-assets-dev
+    acl: public_read
+    cache_control: "max-age=300"
+    local-dir: dist
+    skip_cleanup: true
+    on:
+      branch: master
+after_deploy:
+  - if [ "$TRAVIS_BRANCH" == "master" ]; then aws cloudfront create-invalidation --distribution-id E51L08TW3241I --paths "/*"; fi
```

Comment:
Wrong branch for this work

Reply:
Updated the  [ "$TRAVIS_BRANCH" == "ep-upgrade" ]